card/mod/standard/set/all/rich_html/form.rb

```diff
@@ -140,30 +140,50 @@ def process_edit_fields fields
       field_nest field, opts
     end.join "\n"
   end
+  ###
 
-  def form_for_multi
-    instantiate_builder(form_for_multi_input_name, card, {})
+  # If you use subfield cards to render a form for a new card
+  # then the subfield cards should be created on the new card not the existing
+  # card that build the form
+
+
+  def form
+    @form ||= begin
+      @form_root = true unless parent&.form_root
+      instantiate_builder(form_prefix, card, {})
```

Comment:
I prefer `form_prefix` to `input name`, because the complete index name has the field name in it, and I think it might have a slightly broader use than just inputs (?).  But we could change this, eg, to `input_prefix` or `input_name_prefix`